### PR TITLE
Adjust to changes in ERTAPI

### PR DESCRIPTION
**Pull request by oysteoh**
Changes made in the datamodel in ERTAPI - this PR depends on https://github.com/equinor/ert/pull/1170

### CODE REVIEW

**Comment by oysteoh** [comment]
By checking out and using the referenced PR the reviewer should now be able to run this PR without issues visualizing the data from ERT

File: ertviz/models/realization.py [hunk 1]
```diff
@@ -1,18 +1,4 @@
...
-    def data(self):
-        if self._data is None:
-            self._data = get_data(self._data_url)
-        return self._data
-
-    @property
-    def name(self):
-        return self._name
+        self.name = realization_schema["name"]
+        self.data = realization_schema["data"]
```
Comment (ManInFez) [inline ertviz/models/realization.py:4]: if the `realization_schema["data"]` can be both an array of data and a url, how will the actual data be retrieved when it is a URL?

Comment (oysteoh) [reply ertviz/models/realization.py:4]: 🤔  you probably have a point, but i don't get it...? can it be both? I think we are only sending "data" now along with the values?

Comment (ManInFez) [reply ertviz/models/realization.py:4]: in the test data set it seems to be urls

Comment (oysteoh) [reply ertviz/models/realization.py:4]: hmm.... where exactly? I think i've changed all - but i'm might be mistaken here?

Comment (oysteoh) [reply ertviz/models/realization.py:4]: forget it - now i see! 👍  will fix!

File: tests/data/snake_oil_data.py [hunk 2]
```diff
@@ -140,41 +140,41 @@ def content(self):
...
+        "axis": {"data": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]},
         "ensemble_id": "1",
         "name": "SNAKE_OIL_GPR_DIFF",
         "realizations": [
             {
-                "data_url": "http://127.0.0.1:5000/data/284",
+                "data": "http://127.0.0.1:5000/data/284",
                 "name": 0,
                 "ref_url": "http://127.0.0.1:5000/ensembles/1/realizations/0",
                 "summarized_misfits": {},
```
Comment (ManInFez) [inline tests/data/snake_oil_data.py:180]: data is here a URL

Comment (oysteoh) [reply tests/data/snake_oil_data.py]: Yes i forgot to change the values on the data not being used ... fixed for those used in the test... Now i replaced all the urls with just dummy data - i'm not sure how to approach this with a sane test data set - but that we can discuss later 👍

**Review by ManInFez — APPROVED** [review]
LGTM!

**Pull request merged**